### PR TITLE
(#3367) - Give notice of local_se deprecation

### PR DESCRIPTION
**Pull request by daleharvey**

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -671,6 +671,12 @@ exports.explain404 = function (str) {
   }
 };
 
+exports.info = function (str) {
+  if (process.browser && 'console' in global && 'info' in console) {
```
Comment (nolanlawson) [inline lib/utils.js]: this is probably not safe for node-webkit. you could do `typeof console !=== 'undefined'` instead of `'console' in global`

**Comment by nolanlawson** [comment]
+1 after nit

**Comment by daleharvey** [comment]
heh wow, green in ie + iphone without restarts

**Comment by daleharvey** [comment]
Fixed nit and green, thanks

https://github.com/pouchdb/pouchdb/commit/da13c71ab8e965e3c0efe3c6f71b975fb05cba93

**Pull request closed without merging**